opm/autodiff/SimulatorBase_impl.hpp

```diff
@@ -110,14 +110,31 @@ namespace Opm
         }
 
         // init output writer
-        output_writer_.writeInit( timer, geo_.nnc() );
+        output_writer_.writeInit( geo_.nnc() );
 
         std::string restorefilename = param_.getDefault("restorefile", std::string("") );
         if( ! restorefilename.empty() )
         {
             // -1 means that we'll take the last report step that was written
             const int desiredRestoreStep = param_.getDefault("restorestep", int(-1) );
-            output_writer_.restore( timer, state, prev_well_state, restorefilename, desiredRestoreStep );
+
+            WellsManager wells_manager(eclipse_state_,
+                    desiredRestoreStep,
```

Comment:
Strange indentation, tool-dependent? I'd expect this to start at the same column as eclipse_state_.